source/views/dictionary/list.js

```diff
@@ -14,16 +14,25 @@ import {
   ListSectionHeader,
   ListSeparator,
 } from '../components/list'
+import LoadingView from '../components/loading'
 import type {WordType} from './types'
 import {tracker} from '../../analytics'
 import groupBy from 'lodash/groupBy'
 import head from 'lodash/head'
+import uniq from 'lodash/uniq'
+import words from 'lodash/words'
+import deburr from 'lodash/deburr'
+import isString from 'lodash/isString'
 import {data as terms} from '../../../docs/dictionary.json'
+import {SearchBar} from '../components/searchbar'
 
 const rowHeight = Platform.OS === 'ios' ? 76 : 89
 const headerHeight = Platform.OS === 'ios' ? 33 : 41
 
 const styles = StyleSheet.create({
+  wrapper: {
```

Comment:
Necessary to have both the searchbar and the listview both render. Without the flex, the list disappears.